packages/core/src/main/logger/ipc-file-logger.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import type { LogEntry, transports } from "winston";
+
+type IpcFileLoggerOptions = Omit<transports.FileTransportOptions, "filename">;
+
+class IpcFileLogger {
```

Comment:
Don't quite see why this needs to be a class, could just be created within the `ipc-file-logger.injectable.ts` file. Especially since there will never bee a need to `extend` this class.